gigantum/gigantum-client
# Update disk space check to use root of fs due to docker changes

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
This small pr changes the location checked in the `df` call to be the root file system `/`. Due to changes in docker (at least on mac) and some unique VM configurations we've started to see, this check can erroneously report more storage that what is actually available. 

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
- running `/api/sysinfo` should properly report how much storage is available to docker's overlay file system on mac.
- Verify on widows hyperv/wsl2 this is also reported correctly
- Verify on linux the available space is reported correctly.
